pyscf/tdscf/uhf.py

```diff
@@ -308,6 +394,193 @@ def add_hf_(a, b, hyb=1):
 
     return a, b
 
+def get_ab_sf(mf, mo_energy=None, mo_coeff=None, mo_occ=None,collinear_samples=200):
```

Comment:
why do you need an additional setting to `collinear_samples`? Can it use just the one from `mf._numint.collinear_samples` ?

Reply:
There are two reasons for that. The first one, mf is a UKS object not a multicollinear object and its _numint doesn't contain 'collinear_samples' features The second one, I want get_ab_sf() function can serve as a independent function to provide the A and B matrix of spin flip TDDFT/ TDA.